Gemfile

```diff
@@ -16,6 +16,7 @@ gem 'jbuilder', '~> 2.5'
 
 
 group :development, :test do
+  gem 'devise'
```

Comment:
This gem should go in the production group

Reply:
Only? Or can we use that gem in 'general' group?